### ISSUE TITLE
Calicoctl commands probably don't need ipv4|ipv6 at the start of many of them

### ISSUE THREAD

**Issue by tomdee** [Metaswitch/calico-docker#102]
It can be inferred from the IP.

**Comment by robbrockbank** [comment]
There are a few commands that have the ipv4|ipv6 options

  calicoctl (ipv4|ipv6) pool (add|remove) <CIDR>
  calicoctl (ipv4|ipv6) pool show
  calicoctl (ipv4|ipv6) bgppeer rr (add|remove) <IP>
  calicoctl (ipv4|ipv6) bgppeer rr show
  calicoctl (ipv4|ipv6) container <CONTAINER> (add|remove) <IP> [--interface=<INTERFACE>]

The configure commands we can infer the ip version, but the show commands we cannot.  A few alternative suggestions for simplifying:
-  Remove the ipv4|ipv6 from all commands.  For the show commands, show both ipv4 and ipv6 where appropriate
-  Have the ipv4|ipv6 as optional on the configure commands
-  Remove the ipv4|ipv6 from the configure commands and leave the show commands as they are (seems a bit nasty as the format for teh show/configure is slightly different)
-  Remove the ipv4|ipv6 from the configure commands and move the ipv4/ipv6 option to the end of the command (e.g. calicoctl pool show ipv4)

**Comment by Symmetric** [comment]
I like a combination of 1) and 4) -- remove ipvX as a mandatory parameter
from all commands, add as an optional flag at the end of all the show
commands.

**Comment by spikecurtis** [comment]
I like the ip2 commands as a model.  They accept -4 or -6 as arguments, but default to IPv4 without qualification.

http://linux.die.net/man/8/ip

**Comment by tomdee** [comment]
So - we can probably get away with removing ipv4/ipv6 everywhere (even show commands) and showing both ipv4 and v6 in the show commands
...but if we can keep (-4/-6) as an option on the show commands then that would be nice - so lets try to do that.

**Comment by robbrockbank** [comment]
Fixed in pull req #110 pending review.

I removed ipv4/ipv6 from all configure commands, added as an optional --ipv4 | --ipv6 to the show commands, where the default is to show both.  

I also changed the syntax slightly of the add additional ip commands since the syntax was extremely similar to the adding a container, just reordered.

To add an ip, it's now:

  calicoctl container <CONTAINER> ip (add|remove) <IP> [--interface=<INTERFACE>]

i.e. I've added the "ip" before the add|remove.
